types/block.go

```diff
@@ -34,8 +40,10 @@ const (
 	// Uvarint length of MaxBlockSizeBytes: 4 bytes
 	// 2 fields (2 embedded):               2 bytes
 	// Uvarint length of Data.Txs:          4 bytes
-	// Data.Txs field:                      1 byte
-	MaxOverheadForBlock int64 = 11
+	// Data fields:                         6 bytes
+	// Hash in Data                        32 bytes
```

Comment:
Isn't the overhead about protobuf overhead only? If so, how come we count the hash in data?

Reply:
good point, added another var to hopefully make the distinction clearer

https://github.com/celestiaorg/celestia-core/pull/737/commits/d3aadefb1fc234c06f0872967307847dfdd3b93c